lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,8 +4,20 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
+CreateConVar("wire_cpu_max_frequency", "1400000", {FCVAR_REPLICATED})
+
 if CLIENT then return end -- No more client
 
+cpu_max_frequency = nil
```

Comment:
for the scope of the variable, i agree.
but for the variable itself, my way is easier for me because i do not have to remember that this value is not a number and i don't need to have this GetInt() each time i call it , for myself , a programmer simplify is task as much as possible
modifying the scope , yes, the value itself , no